examples/driveoutput.rs

```diff
@@ -27,7 +28,9 @@ fn do_main(args: Cli) -> errors::Result<()> {
 
     // NOTE: we set the default value to the desired state so
     // setting it separately is not required
-    let _handle = chip.get_line(args.line)?.request(RequestFlags::OUTPUT, args.value, "readinput")?;
+    let _handle =
+        chip.get_line(args.line)?
```

Comment:
Just an observation, in `blinky.rs` on line 35 you have this exact same get_line, request going on but format it slightly differently. Don't know how much this sort of consistency matters to you

Reply:
That is interesting.  This was all just formatted using `cargo fmt`.  Would have to dig into how it works to figure out why it handled those differently (maybe something related to length with the `_` in one of them).